modules/ebpf/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+module_switch(ENABLE_EBPF "Enable ebpf module (requires ebpf toolchain)" LIBBPF_FOUND)
+
+if (NOT ENABLE_EBPF)
+  return()
+endif()
+
+set(BPF_CC clang)
+set(LIBBPF_INCLUDE /usr/include)
+set(BPF_CFLAGS -nostdinc -isystem `${BPF_CC} -print-file-name=include` -target bpf -I${CMAKE_CURRENT_BINARY_DIR} -I${LIBBPF_INCLUDE} -fPIC -O2 -g)
+set(BPFTOOL bpftool)
```

Comment:
I think we can do better detection, e.g. with `find_program()` and `pkg_get_variable()`. I have written a `FindLIBBPF.cmake` helper script to do so. Can you check it out?

Reply:
oops, I didn't notice while fixing up the previous note.

Reply:
I have merged this into this branch now.

Reply:
btw, I have removed the compilation of random.skel.c as a unit. It is included as a whole file into ebpf-reuseport.c This is the same as our autoconf based build operates.

Reply:
Nice, thanks! :)